angular/core/models/membership_model.coffee

```diff
@@ -4,7 +4,7 @@ angular.module('loomioApp').factory 'MembershipModel', (BaseModel, AppConfig) ->
     @plural: 'memberships'
     @indices: ['id', 'userId', 'groupId']
     @searchableFields: ['userName', 'userUsername']
-    @serializableAttributes: AppConfig.permittedParams.membership
```

Comment:
We need to move this into permittedParams.membership (in `permitted_params.rb` on the server), rather than replacing it on the client. :red_circle: